src/java/generator/MapGenerator.java

```diff
@@ -8,10 +10,13 @@
 import util.Pipeline;
 
 import java.awt.*;
+import java.io.File;
```

Comment:
remove file import (you should optimize imports for the entire project, intellij has a feature for that, ctrl+alt+o, I don't know if that works on the entire src dir though, you can check a box when commiting)